### DISCUSSION TITLE
Support Question: Is this needed for Ubuntu 22

### DISCUSSION THREAD

**Discussion by worthwhileindustries** [wslutilities/wslu#248] [Q&A]
This isn't a bug but, more of a support question. I am doing a release upgrade from ubuntu 20 to 22 in and noticed this package isn't supported anymore. Is this package not needed with the new version or it's just not supported yet?

`No longer supported: wslu`

Default Distribution: Ubuntu
Default Version: 2
WSL version: 0.66.2.0
Kernel version: 5.15.57.1
WSLg version: 1.0.42
MSRDC version: 1.2.3401
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.521

**Comment by car-meows** [comment]
wslu is no longer included by default by Ubuntu starting from Ubuntu 22.04. Also, for future support question, use **Discussion** instead of the issue, thank you!
